src/app/auth/guards/is-logged-in.guard.ts

```diff
@@ -27,7 +27,7 @@ export class IsLoggedInGuard implements CanActivate {
     route: ActivatedRouteSnapshot,
     state: RouterStateSnapshot):
     Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
```

Comment:
@k1k8r you don't need to specify union of types since your guard returns just boolean type. Will be better to keep just one type.